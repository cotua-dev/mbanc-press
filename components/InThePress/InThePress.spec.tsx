import { cleanup, render, RenderResult } from '@testing-library/react';
import { InThePress } from './index';

describe('InThePress Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<InThePress/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll(cleanup);
});
