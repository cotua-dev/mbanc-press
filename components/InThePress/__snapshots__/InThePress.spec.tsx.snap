// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InThePress Component renders 1`] = `
<section
  class="in-the-press"
>
  <div
    class="jumbotron"
  >
    <picture>
      <source
        srcset="/images/press/press_m.svg"
      />
      <img
        alt="Mbanc Press Logo"
        class="press-logo"
        loading="lazy"
        src="/images/press/press_m.svg"
      />
    </picture>
  </div>
  <div
    class="press-header"
  >
    <picture>
      <source
        srcset="/images/press/press.svg"
      />
      <img
        alt="Mbanc Press Icon"
        class="press-header-logo"
        loading="lazy"
        src="/images/press/press.svg"
      />
    </picture>
    <h1
      class="title"
    >
      <span>
        MBANC
      </span>
       In The Press
    </h1>
  </div>
  <iframe
    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
    allowfullscreen=""
    class="aspect-video mb-12 sm:mb-20 w-full sm:w-2/3"
    frameborder="0"
    src="https://www.youtube-nocookie.com/embed/59ssQjGAvW4?controls=0&start=695"
    title="YouTube video player"
  />
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/thewashingtonpost_color.svg"
      />
      <img
        alt="The Washington Post Logo"
        class="news-logo "
        height="46"
        loading="lazy"
        src="/images/press/new/thewashingtonpost_color.svg"
        width="300"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.washingtonpost.com/business/2021/08/19/how-qualify-mortgage-with-uneven-income"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          What we have noticed is the performance of these loans are better than your traditional conventional loans.
        </strong>
        <p
          class="remainder"
        >
          As the mortgage servicers who deal in non-QM loans establish positive performance trends, non-QM loan programs will continue to expand.
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.washingtonpost.com/business/2021/08/19/how-qualify-mortgage-with-uneven-income"
      rel="noreferrer"
      target="_blank"
    >
      The Washington Post
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/yahoo_color.svg"
      />
      <img
        alt="Yahoo Logo"
        class="news-logo "
        height="55"
        loading="lazy"
        src="/images/press/new/yahoo_color.svg"
        width="200"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://finance.yahoo.com/finance/news/prepare-housing-market-crash-2021-120051439.html"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          What’s different today from what we saw in 2008
        </strong>
        <p
          class="remainder"
        >
          is that people who are qualifying for loans are actually qualified. They are creditworthy.
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://finance.yahoo.com/finance/news/prepare-housing-market-crash-2021-120051439.html"
      rel="noreferrer"
      target="_blank"
    >
      Yahoo!
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/usatoday_color.svg"
      />
      <img
        alt="USA Today Logo"
        class="news-logo usa-sun-fix"
        height="70"
        loading="lazy"
        src="/images/press/new/usatoday_color.svg"
        width="300"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.usatoday.com/story/sponsor-story/mbanc/2021/03/22/how-unconventional-approach-mortgages-helped-two-entrepreneurs/4725567001"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          MBANC has a different view on how you should qualify for a mortgage.
        </strong>
        <p
          class="remainder"
        >
          The lender personalizes the qualification process and looks at you holistically.
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.usatoday.com/story/sponsor-story/mbanc/2021/03/22/how-unconventional-approach-mortgages-helped-two-entrepreneurs/4725567001"
      rel="noreferrer"
      target="_blank"
    >
      USA Today
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/thesun_color.svg"
      />
      <img
        alt="The Sun Logo"
        class="news-logo usa-sun-fix"
        height="87"
        loading="lazy"
        src="/images/press/new/thesun_color.svg"
        width="200"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.the-sun.com/money/3288694/how-pay-off-mortgage-years-early"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          Although it might feel good to know you fully own your home,
        </strong>
        <p
          class="remainder"
        >
          there might be more financially savvy places to put your money.
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.the-sun.com/money/3288694/how-pay-off-mortgage-years-early"
      rel="noreferrer"
      target="_blank"
    >
      The Sun
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/usatoday_color.svg"
      />
      <img
        alt="USA Today Logo"
        class="news-logo usa-sun-fix"
        height="70"
        loading="lazy"
        src="/images/press/new/usatoday_color.svg"
        width="300"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.usatoday.com/story/sponsor-story/mbanc/2021/02/18/self-employed-and-want-buy-home-process-easier-than-expected/6769814002"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          Because MBANC is funded by private investors looking for entrepreneurs to whom they can lend,
        </strong>
        <p
          class="remainder"
        >
          it controls all of its loan analysis and decision-making.
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.usatoday.com/story/sponsor-story/mbanc/2021/02/18/self-employed-and-want-buy-home-process-easier-than-expected/6769814002"
      rel="noreferrer"
      target="_blank"
    >
      USA Today
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/nationalmortgagenews_color.svg"
      />
      <img
        alt="National Mortgage News Logo"
        class="news-logo usa-sun-fix"
        height="25"
        loading="lazy"
        src="/images/press/new/nationalmortgagenews_color.svg"
        width="300"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.nationalmortgagenews.com/news/non-qm-lender-mbanc-underwrites-borrower-income-from-mobile-wallets"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          Mbanc is a direct-to-consumer non-qualified mortgage lender
        </strong>
        <p
          class="remainder"
        >
          whose customer base includes business owners who accept online payments for their products and services.
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.nationalmortgagenews.com/news/non-qm-lender-mbanc-underwrites-borrower-income-from-mobile-wallets"
      rel="noreferrer"
      target="_blank"
    >
      National Mortgage News
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/bankrate_color.svg"
      />
      <img
        alt="Bankrate News Logo"
        class="news-logo "
        height="60"
        loading="lazy"
        src="/images/press/new/bankrate_color.svg"
        width="140"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.bankrate.com/mortgages/how-to-buy-a-house-with-low-income/"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          “There are a few other strings attached, too: If you are a first-time buyer,
        </strong>
        <p
          class="remainder"
        >
          you will be required to go through a homeownership education program,” adds Tabitha Mazzara
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.bankrate.com/mortgages/how-to-buy-a-house-with-low-income/"
      rel="noreferrer"
      target="_blank"
    >
      Bankrate
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/gobankingrates_color.svg"
      />
      <img
        alt="GoBankingRates News Logo"
        class="news-logo "
        height="45"
        loading="lazy"
        src="/images/press/new/gobankingrates_color.svg"
        width="180"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.gobankingrates.com/loans/mortgage/pros-cons-online-mortgage-lenders/"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          We’ve made it easy to do everything from your couch
        </strong>
        <p
          class="remainder"
        />
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.gobankingrates.com/loans/mortgage/pros-cons-online-mortgage-lenders/"
      rel="noreferrer"
      target="_blank"
    >
      GoBankingRates
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/msn_color.svg"
      />
      <img
        alt="MSN Logo"
        class="news-logo "
        height="45"
        loading="lazy"
        src="/images/press/new/msn_color.svg"
        width="100"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.msn.com/en-us/money/realestate/how-to-buy-a-house-with-a-low-income/ar-AAPHtZ2"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          There are a few other strings attached, too:
        </strong>
        <p
          class="remainder"
        >
          If you are a first-time buyer, you will be required to go through a homeownership education program," adds Tabitha Mazzara, director of operations at MBANC, a mortgage lender.
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.msn.com/en-us/money/realestate/how-to-buy-a-house-with-a-low-income/ar-AAPHtZ2"
      rel="noreferrer"
      target="_blank"
    >
      MSN
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/lendingtree_color.svg"
      />
      <img
        alt="LendingTree News Logo"
        class="news-logo "
        height="65"
        loading="lazy"
        src="/images/press/new/lendingtree_color.svg"
        width="180"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.lendingtree.com/home/mortgage/what-is-a-balloon-mortgage-loan"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          I think that they would be solely for a savvy real estate investor
        </strong>
        <p
          class="remainder"
        >
          that understands how to move through the market and is comfortable with the terms within the balloon note
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.lendingtree.com/home/mortgage/what-is-a-balloon-mortgage-loan"
      rel="noreferrer"
      target="_blank"
    >
      LendingTree
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/dailynationtoday_color.svg"
      />
      <img
        alt="Daily Nation Today Logo"
        class="news-logo "
        height="42"
        loading="lazy"
        src="/images/press/new/dailynationtoday_color.svg"
        width="200"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://dailynationtoday.com/how-does-a-wraparound-mortgage-work"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          The vendor will get the clear advantage
        </strong>
        <p
          class="remainder"
        >
          of the ability to cost an excessive fee of curiosity for a mortgage
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://dailynationtoday.com/how-does-a-wraparound-mortgage-work"
      rel="noreferrer"
      target="_blank"
    >
      Daily Nation Today
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/businessinsider_color.svg"
      />
      <img
        alt="Business Insider Logo"
        class="news-logo "
        height="47"
        loading="lazy"
        src="/images/press/new/businessinsider_color.svg"
        width="150"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.businessinsider.com/what-is-underwriting"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          "Underwriting is basically just verifying all of the information the borrower has provided on their loan application
        </strong>
        <p
          class="remainder"
        >
          — their income, their tax returns, their bank accounts and other assets," says Mayer Dallal
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.businessinsider.com/what-is-underwriting"
      rel="noreferrer"
      target="_blank"
    >
      Business Insider
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/gobankingrates_color.svg"
      />
      <img
        alt="GoBankingRates News Logo"
        class="news-logo "
        height="45"
        loading="lazy"
        src="/images/press/new/gobankingrates_color.svg"
        width="180"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.gobankingrates.com/investing/real-estate/why-holidays-are-good-time-to-buy-house"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          “If someone is selling during the holidays, it usually indicates they've already left their former home behind,
        </strong>
        <p
          class="remainder"
        >
          and they're spending the holidays elsewhere, so they have emotionally disassociated themselves from the home,” Mayer said
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.gobankingrates.com/investing/real-estate/why-holidays-are-good-time-to-buy-house"
      rel="noreferrer"
      target="_blank"
    >
      GoBankingRates
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/cnbc_color.svg"
      />
      <img
        alt="CNBC Logo"
        class="news-logo "
        height="28"
        loading="lazy"
        src="/images/press/new/cnbc_color.svg"
        width="180"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.cnbc.com/2021/12/02/aiming-to-buy-a-home-in-the-next-year-do-these-4-things.html"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          “It's tough to overstate the importance of credit scores to a mortgage loan application,”
        </strong>
        <p
          class="remainder"
        >
          says Tabitha Mazzara, director of operations at Mortgage Bank of California. “They're one of the top things lenders consider.”
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.cnbc.com/2021/12/02/aiming-to-buy-a-home-in-the-next-year-do-these-4-things.html"
      rel="noreferrer"
      target="_blank"
    >
      CNBC
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/nextadvisor_color.svg"
      />
      <img
        alt="NextAdvisor Logo"
        class="news-logo "
        height="46"
        loading="lazy"
        src="/images/press/new/nextadvisor_color.svg"
        width="200"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://time.com/nextadvisor/mortgages/energy-efficient-mortgages/"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          Business owners and homeowners are turning to financing options like green loans
        </strong>
        <p
          class="remainder"
        >
          or energy efficient mortgages so they can make eco-friendly and energy-saving improvements while saving money on their utility bills every month.”
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://time.com/nextadvisor/mortgages/energy-efficient-mortgages/"
      rel="noreferrer"
      target="_blank"
    >
      NextAdvisor
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/msn_color.svg"
      />
      <img
        alt="MSN Logo"
        class="news-logo "
        height="45"
        loading="lazy"
        src="/images/press/new/msn_color.svg"
        width="100"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.msn.com/en-us/money/realestate/the-number-of-us-cities-with-an-average-home-price-of-dollar1-million-tripled-last-year/ar-AAUd0Mt"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          Look at any open house — they're crazy crowded.
        </strong>
        <p
          class="remainder"
        >
          The competition and demand is so high that even if you have government help, someone else might come in with 50% down.
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.msn.com/en-us/money/realestate/the-number-of-us-cities-with-an-average-home-price-of-dollar1-million-tripled-last-year/ar-AAUd0Mt"
      rel="noreferrer"
      target="_blank"
    >
      MSN
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/forbes_color.svg"
      />
      <img
        alt="Forbes Logo"
        class="news-logo "
        height="32"
        loading="lazy"
        src="/images/press/new/forbes_color.svg"
        width="131"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://www.forbes.com/advisor/mortgages/homes-for-sale-hit-record-low"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          It's a true sellers' market right now,
        </strong>
        <p
          class="remainder"
        >
          with more adherence to sellers' demands than normal.
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://www.forbes.com/advisor/mortgages/homes-for-sale-hit-record-low"
      rel="noreferrer"
      target="_blank"
    >
      Forbes
    </a>
  </div>
  <hr
    class="separator"
  />
  <div
    class="quote-wrapper"
  >
    <picture>
      <source
        srcset="/images/press/new/money_color.svg"
      />
      <img
        alt="Money Logo"
        class="news-logo "
        height="45"
        loading="lazy"
        src="/images/press/new/money_color.svg"
        width="130"
      />
    </picture>
    <div
      class="entire-quote"
    >
      <span
        class="quote"
      >
        “
      </span>
      <blockquote
        cite="https://money.com/failed-homebuyers-return-trend-tips"
        class="quote"
      >
        <strong
          class="catch-phrase"
        >
          "They're going to have to up their budgets," says Mayer Dallal, managing director at mortgage lender MBANC.
        </strong>
        <p
          class="remainder"
        >
          "Interest rates are going to keep creeping up, and home prices may either continue to rise or may flatten, but I don't see them going down anytime soon. A budget that was reasonable a couple of months ago may not be as interest rates climb."
        </p>
      </blockquote>
    </div>
    <a
      class="quote-link"
      href="https://money.com/failed-homebuyers-return-trend-tips"
      rel="noreferrer"
      target="_blank"
    >
      Money
    </a>
  </div>
</section>
`;
